Purchases/Purchasing/StoreKitAbstractions/SubscriptionPeriod.swift

```diff
@@ -20,6 +20,8 @@ import StoreKit
     public let unit: PeriodUnit
 
     init(value: Int, unit: PeriodUnit) {
+        assert(value >= 0, "Invalid value: \(value)")
```

Comment:
This isn't compiled in release mode, but I think it's useful for debugging?
Not sure if we should check that it's also not `0`.

Reply:
Sounds good. I can't think of a use case where it'd be valid to have `0`, so we might as well check for that too

Reply:
Will do 👍